<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputTypeDialog</class>
 <widget class="QDialog" name="InputTypeDialog">
  <property name="windowTitle">
   <string>New Input</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="localRadio">
     <property name="text">
      <string>Local File</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="localLayout">
     <item>
      <widget class="QLineEdit" name="localFilePath"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="overpassRadio">
     <property name="text">
      <string>Overpass</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InputTypeDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InputTypeDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
